docs/supported-libraries.md

```diff
@@ -17,111 +17,128 @@ or [contributing](../CONTRIBUTING.md).
+| [Apache Struts 2](https://github.com/apache/struts)                                                                               | 2.3+                          | TODO                                                                                   |
+| [Apache Tapestry](https://tapestry.apache.org/)                                                                                   | 5.4+                          | TODO                                                                                   |
+| [Apache Wicket](https://wicket.apache.org/)                                                                                       | 8.0+                          | TODO                                                                                   |
+| [Armeria](https://armeria.dev)                                                                                                    | 1.3+                          | TODO                                                                                   |
+| [AsyncHttpClient](https://github.com/AsyncHttpClient/async-http-client)                                                           | 1.9+                          | TODO                                                                                   |
+| [AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html)                                                      | 1.0+                          | TODO                                                                                   |
+| [AWS SDK](https://aws.amazon.com/sdk-for-java/)                                                                                   | 1.11.x and 2.2.0+             | TODO                                                                                   |
+| [Azure Core](https://docs.microsoft.com/en-us/java/api/overview/azure/core-readme)                                                | 1.14+                         | TODO                                                                                   |
+| [Cassandra Driver](https://github.com/datastax/java-driver)                                                                       | 3.0+                          | TODO                                                                                   |
+| [Couchbase Client](https://github.com/couchbase/couchbase-java-client)                                                            | 2.0+ and 3.1+                 | TODO                                                                                   |
+| [c3p0](https://github.com/swaldman/c3p0)                                                                                          | 0.9.2                         | [Database Pool Metrics]                                                                |
```

Comment:
This was missing from this page. Curious if it was an intentional omission.

Reply:
not intentional, thx